collaboration_proposal.md

```diff
@@ -0,0 +1,152 @@
+
+    ** Step 2: Merged Simulation Study: **
+        - The final verbal description of the aims and objectives is used for the merged simulation study.
+
+    ** Adversarial Collaboration Techniques: **
+        - Perspective Swapping: Analyzing plans from an adversary's perspective can be used to critically examine the research question from different viewpoints (Kardos & Dexter, 2017).
+        - Core Disagreements: Arrive at clearly defined core disagreements in the research question or objectives that might be the origin for conflicts in successive steps. (Clark et al., 2022)
+    
+2. Specification of Population Model
+    
+    ** Step 1: Individual Simulation Studies: **
+        - Jointly agree on core aspects of a population model wihtout explicit specification of the model, e.g. Confirmatory Factor Analysis. (This might be implicit in the verbal description of the aims and objectives)
+        - Then for each Study independently specify the remaining aspects of the population model, e.g. number of factors, number of indicators, factor loadings, error variances, etc.
+
+    ** Step 2: Merged Simulation Study: **
+    - One population model as a result of the adversarial process.
+
+    ** Adversarial Collaboration Techniques used: **
+    - Perspective Swapping, Core Disagreements
+    - Listing: List, question, and categorize assumptions (Kardos & Dexter, 2017):
+        - to ensure the population model is robust and well-founded
+        - finegrained discussion of each aspect of the model weighing evidence for each decision.
+        - Label each decision for a specific model aspect as either evidence-based (on exisiting literature) (i.e. "We chose this factor strucutre because it is evidently common (source x,y, ...)") or pragmatically assumption based due to absence of distinctive evidence (i.e. We chose this factor structure because we believe it is the most representiative...).
+
+3. Experimental Design Simulation Procedures
+
+    ** Step 1: Individual Simulation Studies: **
+        - Chosen independently by each collaborator.
+
+    ** Step 2: Merged Simulation Study: **
+        - Generate one simulation design as a result of the adversarial process.
+    
+    ** Adversarial Collaboration Techniques used: **
+        - Perspective Swapping, Core Disagreements, Listing
+
+4. Data Generation Mechanism
+    ** Step 1: Individual Simulation Studies: **
+        - Chosen independently by each collaborator.
+
+    ** Step 2: Merged Simulation Study: **
+        - result of the adversarial process.
+    
+    ** Adversarial Collaboration Techniques used: **
+        - Perspective Swapping, Core Disagreements, Listing
+        - Red-Teaming: what if scenarios & counter arguments (Kardos & Dexter, 2017):
+            - Generate and test alternative data generation mechanisms under which the arbiters method fails or underperforms.
```

Comment:
A generell comment on your inclusion of Red-Teaming in this proposal: I really like the idea of actively trying to push the limits of assumptions/ scenarios etc. However, I am worried about multiple things:
1. Multiple testing: Not only in the sense of p-values but also more general (Is this a problem in simulation studies?)
2. Loss of evidence base: Arguing mainly on very specific scenarios that one constructs might not be fruitful.
3. Complexity: Having each step that you propose to Red-Team at included individually might lead to overcomplexity.
4. No clear distinction between simulation protocol and execution

I suggest, analogously to my proposal, that we have a clear seperation between a "preregistration"/simulation protocol as suggested by Morris et al. (2019), and the execution. In the simulation protocol, we argue mainly on an evidence base and on substantive questions, while in the execution, we focus on Red-Teaming specific scenarios under which the others methodology/ hypotheses fail.

Reply:
Another thought with regards to the explicitness of the Red-Teaming aspects you propose: I think that for many decisions, our main focus should be on reasoning about the superiority of the own method, not just the break down of the other. In that sense, I think that besides Red-Teaming, it will also be important to us to defend our own method. Your proposal focusses mainly on the Red-Teaming aspect, but maybe you agree with me here, in that sense that Red-Teaming always implies a defense as well...

Reply:
I agree and this defence would ideally be the result of the attack, as a defence is best presented and tested in response to an attack.